### ISSUE TITLE
JSONDecodeError: Expecting value: line 1 column 1 (char 0)

### ISSUE THREAD

**Issue by philfreo** [SiftScience/sift-python#37]
It seems the the Sift client throws JSONDecodeError's sometimes. I have some of these in my log:

```
  File "tasks.py", line 299, in track_sift_event
    response = app.sift_client.track(event_type, attrs)
  File "/home/ubuntu/closeio/venv/local/lib/python2.7/site-packages/sift/client.py", line 105, in track
    return Response(response)
  File "/home/ubuntu/closeio/venv/local/lib/python2.7/site-packages/sift/client.py", line 214, in __init__
    self.body = http_response.json()
  File "/home/ubuntu/closeio/venv/local/lib/python2.7/site-packages/requests/models.py", line 651, in json
    return json.loads(self.text or self.content, **kwargs)
  File "/home/ubuntu/closeio/venv/local/lib/python2.7/site-packages/simplejson/__init__.py", line 516, in loads
    return _default_decoder.decode(s)
  File "/home/ubuntu/closeio/venv/local/lib/python2.7/site-packages/simplejson/decoder.py", line 370, in decode
    obj, end = self.raw_decode(s)
  File "/home/ubuntu/closeio/venv/local/lib/python2.7/site-packages/simplejson/decoder.py", line 400, in raw_decode
    return self.scan_once(s, idx=_w(s, idx).end())
JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by philfreo** [comment]
@fredsadaghiani

**Comment by fredsadaghiani** [comment]
@philfreo what version of the client sdk are you using?

**Comment by philfreo** [comment]
@fredsadaghiani 1.1.2.5

**Comment by philfreo** [comment]
Here's a more recent stack trace:

``` 
 File "/home/ubuntu/closeio/venv/local/lib/python2.7/site-packages/sift/client.py", line 132, in track
    return Response(response)
  File "/home/ubuntu/closeio/venv/local/lib/python2.7/site-packages/sift/client.py", line 256, in __init__
    self.body = http_response.json()
  File "/home/ubuntu/closeio/venv/local/lib/python2.7/site-packages/requests/models.py", line 651, in json
    return json.loads(self.text or self.content, **kwargs)
  File "/home/ubuntu/closeio/venv/local/lib/python2.7/site-packages/simplejson/__init__.py", line 516, in loads
    return _default_decoder.decode(s)
  File "/home/ubuntu/closeio/venv/local/lib/python2.7/site-packages/simplejson/decoder.py", line 370, in decode
    obj, end = self.raw_decode(s)
  File "/home/ubuntu/closeio/venv/local/lib/python2.7/site-packages/simplejson/decoder.py", line 400, in raw_decode
    return self.scan_once(s, idx=_w(s, idx).end())
JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by JohnMcSpedon** [comment]
This should resolved by https://github.com/SiftScience/sift-python/pull/41 (available in version `1.1.2.6`).  Thanks for surfacing this @philfreo! Sorry about the delayed response.
